netstandard/ref/System.Runtime.cs

```diff
@@ -33,6 +33,7 @@ public sealed partial class MemoryFailPoint : System.Runtime.ConstrainedExecutio
     {
         public MemoryFailPoint(int sizeInMegabytes) { }
         public void Dispose() { }
+        ~MemoryFailPoint() { }
```

Comment:
Does this type need a finalizer? That puts more pressure on the GC than needed no?

Reply:
Following what we've done for finalizers, I've excluded this API.